server/protocols/webadmin-cli/IMPLEMENT_WEBADMINCLI.md

```diff
@@ -0,0 +1,48 @@
+# Architecture Decision Record for new James CLI
+
+## Why implement a CLI based on webadmin API
+
+James servers offer a command-line interface in order to interact with the server. However, it relies on the JMX protocol, which is known to be insecure.
+
+Webadmin APIs use HTTP protocol, which is more secure than JMS protocol to interact with James servers.
+
+Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface.  
+
+## Context
+We decided some points about new James CLI architecture:
```

Comment:
We decided to write a new CLI client, running on top of the JVM, communicating with James via the webadmin protocol, using http.